container/containerd/handler.go

```diff
@@ -36,31 +34,21 @@ import (
 )
 
 type containerdContainerHandler struct {
-	client             containerdClient
-	name               string
-	id                 string
-	aliases            []string
 	machineInfoFactory info.MachineInfoFactory
 	// Absolute path to the cgroup hierarchies of this container.
 	// (e.g.: "cpu" -> "/sys/fs/cgroup/cpu/test")
 	cgroupPaths map[string]string
-	// Manager of this container's cgroups.
-	cgroupManager cgroups.Manager
-	fsInfo        fs.FsInfo
-	poolName      string
-	// Time at which this container was created.
-	creationTime time.Time
+	fsInfo      fs.FsInfo
 	// Metadata associated with the container.
-	labels map[string]string
-	envs   map[string]string
-	// The container PID used to switch namespaces as required
-	pid int
+	reference info.ContainerReference
+	envs      map[string]string
+	labels    map[string]string
 	// Image name used for this container.
 	image string
-	// The host root FS to read
-	rootFs string
 	// Filesystem handler.
 	ignoreMetrics container.MetricSet
```

Comment:
We missed removing the `ignoreMetrics` from all handlers

Reply:
wrong line actually, aimed for 48-49

Reply:
you are welcome to do this in your PR.  IIRC ignoreMetrics was still used by the other handlers, but I may be mistaken...